src/applications/caregivers/definitions/UIDefinitions/sharedUI.js

```diff
@@ -76,11 +81,12 @@ export const addressWithoutCountryUI = label => ({
   'ui:title': ' ',
   'ui:order': ['street', 'street2', 'city', 'state', 'postalCode'],
   street: {
-    'ui:title': `${label} current street address`,
-    'ui:errorMessages': { required: 'Please enter a street address' },
+    'ui:title': `${label} current home address`,
+    'ui:description': VeteranHomeAddressDescription,
```

Comment:
Same thing here, as mentioned above. You will need to extend the declaration where it is used and not bake this into the shared schema declaration.